scripts/e2e-email/e2e-email.js

```diff
@@ -0,0 +1,277 @@
+#!/usr/bin/env node
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// NOTE: this script will fail if run against official production
+// auth-server, since `/v1/account/lock` is, of course, not available
+// there.
+
+const assert = require('assert')
+const crypto = require('crypto')
+const url = require('url')
+
+const P = require('../../promise')
+const preq = require('./p-request')
+const localeQuirks = require('./localeQuirks')
+
+var program;
+try {
+  program = require('commander');
+} catch(e) {
+  console.log("This program requires you to do `npm install commander` first.");
+  process.exit(1);
+}
+
```

Comment:
By the way, I did the above since I didn't want to add `commander` as a devDep of this repo. But I could do so, if no one agrees with that choice.